src/node.ts

```diff
@@ -52,6 +52,11 @@ export interface EthereumNodeProps {
      */
   readonly instanceType?: InstanceType;
 
+  /**
+   * The Region in which the node will be created
```

Comment:
This should be down in the class itself, and the comment should be `The Region in which the node exists`

Reply:
I originally had it in the class, but I must have moved it incorrectly when I reset my branch. I moved it and reformatted the comment!